onyo/lib/commands.py

```diff
@@ -87,10 +87,10 @@ def fsck(repo: OnyoRepo,
 
     Parameters
     ----------
-    repo: OnyoRepo
+    repo
         The Repository on which to perform the fsck on.
 
-    tests: list of str, optional
```

Comment:
I think @bpoldrack pointed out a while ago that the meaning of "optional" in docstrings does not 100% align with the meaning of type hints.

IIRC In docstrings it means "is not required to be given" while optional as typehint means "must be given but can be `None`". But if not given, python errors with "argument missing" if not e.g. a default value is specified.

I could imagine that this is not concidered correctly here when auto-generating types?

Reply:
Correct. `Optional[]` means "or type `None`". The use of default values means that the parameter is optional.

In the docs, default values are rendered:
- in the function header
- in the type description of the parameter (I just enabled this)
- in the parameter description (written manually when it deviates from the declared default, such as with mutable types)

Even though that doesn't use the word "optional", I think it's pretty clear what's going on.